src/tests/src/solver/optim/api/testApiOrtoolsLinearProblem.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright 2007-2024, RTE (https://www.rte-france.com)
+ * See AUTHORS.txt
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of Antares-Simulator,
+ * Adequacy and Performance assessment for interconnected energy networks.
+ *
+ * Antares_Simulator is free software: you can redistribute it and/or modify
+ * it under the terms of the Mozilla Public Licence 2.0 as published by
+ * the Mozilla Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Antares_Simulator is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Mozilla Public Licence 2.0 for more details.
+ *
+ * You should have received a copy of the Mozilla Public Licence 2.0
+ * along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
+ */
+#define BOOST_TEST_MODULE test optim api ortools
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <ortools/linear_solver/linear_solver.h>
```

Comment:
Apparently we need `#include <ortools/linear_solver/linear_solver.h>`.
Why is that ?
Indeed, ortools linear solver interface (I mean the dependency interface) should be needed inside the ortools implementation of our problem interface (class **OrtoolsLinearProblem**), but not elsewhere.